### PR TITLE
OpenAPI announcement blog post — Connect LunchBadger/marketing#25 Closes Connect LunchBadger/marketing#25

### PR DESCRIPTION
**Pull request by XVincentX**
The idea would be to spread this blog post a little bit with our social media as well as our Gitter channel and see if we can gather any feedback around.

### CODE REVIEW

File: _posts/2018-08-14-openapi-support-eg.markdown [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+- Connectivity information to a target server
+- The available paths and operations for the API
+- Security mechanisms that can be used across the API
+
+### What can Express Gateway do with an OpenAPI file
+
+We've been exploring the OpenAPI spec for a while and we've been trying to understand what Express Gateway could do
+when such file is provided by the user. In particular, we have identified different touching points
+
+#### Connectivity informations
```
Comment (kevinswiber) [inline _posts/2018-08-14-openapi-support-eg.markdown]: s/informations/information

File: _posts/2018-08-14-openapi-support-eg.markdown [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+We can use the connectivity informations for server target to configure the host of all the [apiEndpoints][apiEndpoints]. In this way we can make sure Express Gateway will receive connections exclusively when received on the specified host.
```
Comment (kevinswiber) [inline _posts/2018-08-14-openapi-support-eg.markdown]: s/informations/information

**Review by kevinswiber — CHANGES_REQUESTED** [review]
@XVincentX This looks great!  Just a couple of letters need to be removed, and I think this is golden.

**Comment by snowcrash007** [comment]
Let me know when it's published and I can get it up on our channels.

**Comment by XVincentX** [comment]
@snowcrash007 I can publish right now.

**Pull request merged**